{
  "cveId": "CVE-2025-53862",
  "eventName": "New CVE Received",
  "cveChangeId": "9642850A-7959-43CB-8F89-9DE46E870FC4",
  "sourceIdentifier": "secalert@redhat.com",
  "created": "2025-07-11T13:15:59.703",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "A flaw was found in Ansible. Three API endpoints are accessible and return verbose, unauthenticated responses. This flaw allows a malicious user to access data that may contain important information."
    },
    {
      "action": "Added",
      "type": "CVSS V3.1",
      "newValue": "AV:A/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N"
    },
    {
      "action": "Added",
      "type": "CWE",
      "newValue": "CWE-497"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://access.redhat.com/security/cve/CVE-2025-53862"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://bugzilla.redhat.com/show_bug.cgi?id=2379359"
    }
  ]
}